upstream-community-operators/oneagent/dynatrace-monitoring.v0.3.1.clusterseviceversion.yaml

```diff
@@ -0,0 +1,274 @@
+    mediatype: "image/png"
+  install:
+    spec:
+      deployments:
+      - name: dynatrace-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: dynatrace-oneagent-operator
+          template:
+            metadata:
+              labels:
+                dynatrace: operator
+                name: dynatrace-oneagent-operator
+                operator: oneagent
+            spec:
+              containers:
+              - command:
+                - dynatrace-oneagent-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please set the `WATCH_NAMESPACE` variable to the `metadata.annotations['olm.targetNamespaces']`:
```
- name: WATCH_NAMESPACE
  valueFrom:
    fieldRef:
      fieldPath: metadata.annotations['olm.targetNamespaces']
```

Reply:
Thanks. I've updated it accordingly.